metric/kamailio5/kamailio.cfg

```diff
@@ -410,41 +420,64 @@ route[STORE] {
 
 
         $var(homer_query_values) = "(10, '" + $utimef(%Y-%m-%d %H:%M:%S) + "', '" + $var(hep_header) +"', '" + $var(ip_header) + "', '" + $var(payload) + "', '" + $msg(buf) + "')";
-        mq_add("queue_sip_insert", "$var(table)", "$var(homer_query_values)");
+
+        if($sht(cc=>packet::count) == $null) $sht(cc=>packet::count) = 0;
+        $sht(cc=>packet::count) = $sht(cc=>packet::count) + 1;
+
+	mq_add("queue_sip_insert", "$var(table)", "$var(homer_query_values)");
```

Comment:
We should rename all var with homer inside because this name brings no value or readability

Reply:
done for homer_query_values and homer_query vars.